cpp/oneapi/dal/backend/primitives/objective_function/logloss.hpp

```diff
@@ -88,4 +88,44 @@ sycl::event compute_hessian(sycl::queue& q,
                             bool fit_intercept = true,
                             const event_vector& deps = {});
 
+template <typename Float>
+sycl::event compute_raw_hessian(sycl::queue& q,
+                                const ndview<Float, 1>& probabilities,
+                                ndview<Float, 1>& out_hessian,
+                                const event_vector& deps = {});
+
+template <typename Float>
+class logloss_hessp {
```

Comment:
```suggestion
class logloss_hessian {
```

or 
```suggestion
class logloss_hessian_primitive {
```
No need for short and unreadable naming

Reply:
hessp is shorten name from hessian product